Add BUILD_STATUS.md to Chrobalt.

briantting opened the pull request.

oxve commented on BUILD_STATUS.md:
I think we should remove broken and obsolete entries. Rows that won't have any badges (workflows that only ran on `main`) we should remove.

The rich diff shows what's broken: https://github.com/youtube/cobalt/pull/4192/files?short_path=c6f3fe6#diff-c6f3fe6286b445ee90f9290fd0470c993b0834f1d558b9abd7beadc98f5eeb20

briantting replied:
done.

oxve commented on BUILD_STATUS.md:
```diff
@@ -0,0 +1,21 @@
+| Raspi-2   | | [![raspi-2_25.lts.1+](https://github.com/youtube/cobalt/actions/workflows/raspi-2_25.lts.1+.yaml/badge.svg?branch=25.lts.1%2B&event=push)](https://github.com/youtube/cobalt/actions/workflows/raspi-2_25.lts.1+.yaml?query=event%3Apush+branch%3A25.lts.1%2B) | [![raspi-2_24.lts.1+](https://github.com/youtube/cobalt/actions/workflows/raspi-2_24.lts.1+.yaml/badge.svg?branch=24.lts.1%2B& ...
+| Stub      | | [![stub_25.lts.1+](https://github.com/youtube/cobalt/actions/workflows/stub_25.lts.1+.yaml/badge.svg?branch=25.lts.1%2B&event=push)](https://github.com/youtube/cobalt/actions/workflows/stub_25.lts.1+.yaml?query=event%3Apush+branch%3A25.lts.1%2B) | [![stub_24.lts.1+](https://github.com/youtube/cobalt/actions/workflows/stub_24.lts.1+.yaml/badge.svg?branch=24.lts.1%2B&event=push)](ht ...
+| Win32     | [![win32](https://github.com/youtube/cobalt/actions/workflows/win32.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt/actions/workflows/win32.yaml?query=event%3Apush+branch%3Amain) | [![win32_25.lts.1+](https://github.com/youtube/cobalt/actions/workflows/win32_25.lts.1+.yaml/badge.svg?branch=25.lts.1%2B&event=push)](https://github.com/youtube/cobalt/actions/w ...
+| Xbox One  | [![xb1](https://github.com/youtube/cobalt/actions/workflows/xb1.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt/actions/workflows/xb1.yaml?query=event%3Apush+branch%3Amain) | | | | | | | | |
```
win32 and xb1 can be removed for trunk. They haven't run in a while. Same below.

dahlstrom-g reviewed (commented):
A few questions about this PR:
* Do we have to add this file in the root?
* Why are the contents so different from before? https://github.com/youtube/cobalt/blob/pre-chrobalt/BUILD_STATUS.md?plain=1
* Isn't it unhygienic to name various branches in a file on the `main` branch?
* Why does it have 7 commits?

oxve commented on BUILD_STATUS.md:
@kaidokert Made a good point about this file, it should probably live under the `chrobalt` directory instead.

oxve commented:
> A few questions about this PR:
> 
> * Do we have to add this file in the root?

It should live under `/chrobalt`.


> * Why are the contents so different from before? [`pre-chrobalt`/BUILD_STATUS.md?plain=1](https://github.com/youtube/cobalt/blob/pre-chrobalt/BUILD_STATUS.md?plain=1&rgh-link-date=2024-10-03T02%3A49%3A42Z)

We're building very different things now. See my comment earlier on the PR.

> * Isn't it unhygienic to name various branches in a file on the `main` branch?

This is the overall build status, it should show all the important builds in one place.

briantting commented:
> A few questions about this PR:
> 
> * Do we have to add this file in the root?
> * Why are the contents so different from before? https://github.com/youtube/cobalt/blob/pre-chrobalt/BUILD_STATUS.md?plain=1
> * Isn't it unhygienic to name various branches in a file on the `main` branch?
> * Why does it have 7 commits?

Added to Chrobalt instead, because added 25 lts and cleaned up main which only does linux and android now, we want this to be a singular status page to monitor everything so kinda but for ease of use its necessary and what we were doing before, because when i hit edit within github thats what happens (its 8 now)
